clients/mobile/src/authentication/login/UI/Login.tsx

```diff
@@ -56,12 +61,14 @@ export default function LoginScreen({ navigation }: LoginScreenProps) {
       <SocialButton
         text="Continue with Google"
         onPress={handleGoogleSignIn}
+        loading={socialLoading === SOCIAL_TYPES.GOOGLE}
         icon={require('../../../../assets/google-icon.png')}
```

Comment:
it'll be better to move the import declaration in the top if possible.

Reply:
updated vai b5f806773e75d95495330c9e409aa203202a15d6